test/e2e/fixtures.go

```diff
@@ -389,6 +389,32 @@ func exportLogs(tb testing.TB, data *TestData, logsSubDir string, writeNodeLogs
 	}); err != nil {
 		tb.Logf("Error when exporting kubelet logs: %v", err)
 	}
+
+	writeVMAgentLog := func(cmd string, targetVMs string) {
+		vms := strings.Split(targetVMs, ",")
+		for _, vm := range vms {
+			tb.Logf("Exporting logs from %s", vm)
+			_, stdout, _, err := data.RunCommandOnNode(vm, cmd)
+			if err != nil {
+				tb.Errorf("Error when exporting antrea-agent logs from %s: %v", vm, err)
+			}
+			w := getNodeWriter(vm, "antrea-agent")
+			if w == nil {
+				// move on to the next VM
+				continue
+			}
+			w.WriteString(stdout)
+			w.Close()
+		}
+	}
+	if testOptions.linVMs != "" {
+		cmd := "sudo cat /var/log/antrea/antrea-agent.log"
```

Comment:
I'd rather not have the command use `sudo`. Notice that the `journalctl` command above doesn't use sudo for example. Maybe we can assume that the user we ssh as, has permissions to read the file? Is that not the case for our testbed?